src/main/java/fi/fmi/avi/util/BulletinHeadingDecoder.java

```diff
@@ -63,7 +68,8 @@ public static BulletinHeading decode(final String input, final ConversionHints h
                 .setGeographicalDesignator(m.group("AA"))//
                 .setBulletinNumber(Integer.parseInt(m.group("ii")))//
                 .setType(type)//
-                .setBulletinAugmentationNumber(Optional.ofNullable(bulletinAugmentationNumber))//
+                .setAugmentationNumber(Optional.ofNullable(bulletinAugmentationNumber))//
```

Comment:
A minor optimization:

```suggestion
                .setNullableAugmentationNumber(bulletinAugmentationNumber)//
```